solana-programs/anchor/audius-data/cli/main.ts

```diff
@@ -146,11 +153,89 @@ async function timeCreateTrack(args: createTrackArgs) {
       let tx = await createTrack({
         program: args.program,
         provider: args.provider,
-        metadata: randomCID(),
-        newTrackKeypair: anchor.web3.Keypair.generate(),
-        userAuthorityKey: userSolKeypair,
-        userStgAccountPDA: options.userStgPubkey,
+        metadata: args.metadata,
+        newTrackKeypair: args.newTrackKeypair,
+        userAuthorityKeypair: args.userAuthorityKeypair,
+        userStgAccountPDA: args.userStgAccountPDA,
+        adminStgPublicKey: args.adminStgPublicKey,
+      });
+      let duration = Date.now() - start;
+      console.log(
+        `Processed ${tx} in ${duration}, user=${options.userStgPubkey}`
+      );
+      return tx;
+    } catch (e) {
+      err = e;
+    }
+  }
+  console.log(err);
+}
+
+async function timeCreatePlaylist(args: createPlaylistArgs) {
+  let retries = 5;
+  let err = null;
+  while (retries > 0) {
+    try {
+      let start = Date.now();
+      let tx = await createPlaylist({
+        program: args.program,
+        provider: args.provider,
+        newPlaylistKeypair: args.newPlaylistKeypair,
+        userStgAccountPDA: args.userStgAccountPDA,
+        userAuthorityKeypair: args.userAuthorityKeypair,
         adminStgPublicKey: args.adminStgPublicKey,
+        metadata: randomCID(),
+      });
+      let duration = Date.now() - start;
+      console.log(
+        `Processed ${tx} in ${duration}, user=${options.userStgPubkey}`
+      );
+      return tx;
+    } catch (e) {
+      err = e;
+    }
+  }
+  console.log(err);
+}
+
+async function timeUpdatePlaylist(args: updatePlaylistArgs) {
+  let retries = 5;
+  let err = null;
+  while (retries > 0) {
+    try {
+      let start = Date.now();
+      let tx = await updatePlaylist({
+        program: args.program,
+        provider: args.provider,
+        playlistPublicKey: args.playlistPublicKey,
+        userStgAccountPDA: args.userStgAccountPDA,
+        userAuthorityKeypair: args.userAuthorityKeypair,
+        metadata: args.metadata,
+      });
+      let duration = Date.now() - start;
+      console.log(
+        `Processed ${tx} in ${duration}, user=${options.userStgPubkey}`
+      );
+      return tx;
+    } catch (e) {
+      err = e;
+    }
+  }
+  console.log(err);
+}
+
+async function timeDeletePlaylist(args: deletePlaylistArgs) {
```

Comment:
it would be really great if we could update the CLI readme as we do this too but can do as a follow-up - not blocking here

Reply:
right. updated